src/_locales/fr.config

```diff
@@ -14,14 +14,14 @@ Marche
 Arrêt
 
 @toggle_current_site
-Basculer le site actuel
+Activer/Désactiver le site actuel
 
 @setup_hotkey_toggle_site
 Configurer la touche de raccourci
 du basculement du site actuel
 
 @toggle_extension
-l'extension bascule
+Activer:Désactiver l'extension
```

Comment:
Maybe "/" should be used same as on line 17.

Reply:
That seems to be a typo.